### PR TITLE
UCLALIB-239: Location CT responsive 

### PR DESCRIPTION
**Pull request by emb03**
- adds .scss compiled to css to hide items
- adjusts panelizer for collapsible menu on mobile (hours)

### CODE REVIEW

File: www/sites/all/themes/uclalib_omega/sass/components/_panelizer-view-mode.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
       // for mobile.
       .pane-linked-map,
       .pane-at-this-location-block-panel-pane-1,
-      .pane-node-field-location-map-image {
+      .pane-node-field-location-map-image.inside {
```
Comment (jameswilson) [inline www/sites/all/themes/uclalib_omega/sass/components/_panelizer-view-mode.scss]: What does this change do?

File: www/sites/all/themes/uclalib_omega/sass/components/_panelizer-view-mode.scss [hunk 2]
```diff
@@ -24,4 +24,17 @@
       }
     }
   }
+  .pane-node-field-location-social-links,
+  .pane-bean-arts-research-guides,
+  .pane-at-this-location-block-panel-pane-1,
+  .pane-about-this-library-panel-pane-1 {
+          display: none;
+  }
```
Comment (jameswilson) [inline www/sites/all/themes/uclalib_omega/sass/components/_panelizer-view-mode.scss]: careful with indentation.

File: www/sites/all/themes/uclalib_omega/sass/components/_panelizer-view-mode.scss (hunk 1 above)
Comment (emb03) [inline www/sites/all/themes/uclalib_omega/sass/components/_panelizer-view-mode.scss]: that must of been a mistake, sometimes phpstorm does that.
will fix

File: www/sites/all/themes/uclalib_omega/sass/components/_panelizer-view-mode.scss (hunk 2 above)
Comment (emb03) [inline www/sites/all/themes/uclalib_omega/sass/components/_panelizer-view-mode.scss]: will fix

File: www/sites/all/modules/features/uclalib_location/uclalib_location.panelizer.inc [hunk 3]
```diff
@@ -773,7 +773,10 @@ function uclalib_location_panelizer_defaults() {
     $pane->style = array(
       'settings' => NULL,
     );
-    $pane->css = array();
+    $pane->css = array(
+      'css_id' => '',
+      'css_class' => '',
+    );
     $pane->extras = array();
```
Comment (jameswilson) [inline www/sites/all/modules/features/uclalib_location/uclalib_location.panelizer.inc]: This hunk is not necessary.

Comment (emb03) [inline www/sites/all/modules/features/uclalib_location/uclalib_location.panelizer.inc]: That hunk is what makes hours collapsible on mobile.

Comment (jameswilson) [inline www/sites/all/modules/features/uclalib_location/uclalib_location.panelizer.inc]: Ok. My understanding was that:

```
$pane->css = array();
```

... and ...

```
    $pane->css = array(
      'css_id' => '',
      'css_class' => '',
    );
```

are functionally equivalent, but just that the later means you loaded the 'CSS Properties' dialog in panels, but didn't actually save any values (ie, the empty strings).  Can you explain what is effectively going on in the rendered HTML when this happens to make the pane collapsible?

Comment (emb03) [inline www/sites/all/modules/features/uclalib_location/uclalib_location.panelizer.inc]: I added: mobile-collapsible  mobile-collapsible--title-hidden-on-desktop

URL:/node/392/panelizer/page_manager/content

If there is a reliable workaround to this problem, please let me know.

Overidden panelizer (I tried adding it to default and nothing happened)

the html renders as:
<div class="panel-pane pane-opening-hours-table mobile-collapsible mobile-collapsible--title-hidden-on-desktop is-mobile mobile-collapsible-processed">

Comment (jameswilson) [inline www/sites/all/modules/features/uclalib_location/uclalib_location.panelizer.inc]: You need to make sure the default is collapsible by ensuring the code contains the following:
```
    $pane->css = array(
      'css_id' => '',
      'css_class' => 'mobile-collapsible mobile-collapsible--title-hidden-on-desktop',
    );
```

Comment (emb03) [inline www/sites/all/modules/features/uclalib_location/uclalib_location.panelizer.inc]: Can I make this change manually in the code?

**Pull request merged**